### ISSUE TITLE
Support numeric conversion of strings with locale-specific grouping & decimal separators

### ISSUE THREAD

**Issue by tfmorris** [OpenRefine/OpenRefine#507]
_Original author: danpaulsmith (December 16, 2011 12:44:17)_

If a figure has a comma in it, e.g. &quot;32,645&quot; - this is parsed as a string instead of an int.

I think any ints or floats containing commas in the form of a string should be parsed as an int/float.

(And as a side note, I've deleted my attachments in &quot;closed&quot; and &quot;fixed&quot; issues, but still have the &quot;Issue attachment storage quota exceeded&quot; message, preventing me from attaching any media to future issues).

Dan

_Original issue: http://code.google.com/p/google-refine/issues/detail?id=507_

**Comment by tfmorris** [comment]
I agree with @dfhuynh that it's dangerous to do this during import, so switching the tag from `Import` to `GREL` as a `toNumber` enhancement request for locale-specific grouping (ie thousands) and decimal separators.

Trifacta's DataWranger [NUMVALUE](https://docs.trifacta.com/display/DP/NUMVALUE+Function) requires them specified separately and explicitly, but I think it'd be useful to be able easily get the defaults for a given user specified locale. It also automatically removes any currency symbol, which I'm not 100% sure is a good idea.

**Comment by thadguidry** [comment]
Talend's support is through both a Format Numbers dialog as well as a function.  https://help.talend.com/r/en-US/Cloud/data-preparation-user-guide/changing-number-format
